### PR TITLE
Action sheet

### PR DESCRIPTION
**Pull request by LeonmanRolls**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@ jobs:
           node-version: ${{ matrix.node-version }}
           cache: 'pnpm'
 
-      - run: pnpm install --frozen-lockfile
+      - run: pnpm install -no-frozen-lockfile
```
Comment (TateB) [inline .github/workflows/main.yml:28]: why is this needed?

File: components/src/components/molecules/Dropdown/Dropdown.tsx [hunk 2]
```diff
@@ -440,58 +501,108 @@ export const Dropdown = ({
...
+          direction,
+          isOpen,
+          setIsOpen,
+          label,
+          items,
+          buttonProps,
+          indicatorColor,
+        }}
       />
+      {match({ responsive, screenSize })
```
Comment (TateB) [inline components/src/components/molecules/Dropdown/Dropdown.tsx:552]: i get why pattern matching is being used here, but not sure that we should use a lib for basic functionality in a single component

**Review by TateB — CHANGES_REQUESTED** [review]

File: .github/workflows/main.yml (hunk 1 above)
Comment (LeonmanRolls) [reply .github/workflows/main.yml:28]: Causes issues for me

File: components/src/components/molecules/Dropdown/Dropdown.tsx (hunk 2 above)
Comment (LeonmanRolls) [reply components/src/components/molecules/Dropdown/Dropdown.tsx:552]: > i get why pattern matching is being used here, but not sure that we should use a lib for basic functionality in a single component

Will be used more widely going forward

Comment (LeonmanRolls) [reply components/src/components/molecules/Dropdown/Dropdown.tsx:552]: It's about 1kb gzipped
